packages/form-core/src/validate/Validator.js

```diff
@@ -75,29 +89,32 @@ export class Validator {
           .config.getMessage}`,
       );
     }
-    return this.constructor.getMessage(composedData);
+    return ctor.getMessage(composedData);
   }
 
   /**
    * @overridable
-   * @param {object} data
-   * @param {*} data.modelValue
-   * @param {string} data.fieldName
-   * @param {*} data.params
-   * @param {string} data.type
-   * @returns {string|Node|Promise<stringOrNode>|() => stringOrNode)}
+   * @param {object} [data]
+   * @param {*} [data.modelValue]
+   * @param {string} [data.fieldName]
+   * @param {*} [data.params]
+   * @param {string} [data.type]
+   * @param {Object.<string,?>} [data.config]
+   * @param {string} [data.name]
+   * @returns {Promise<string|Node>}
    */
-  static async getMessage(/* data */) {
+  // eslint-disable-next-line no-unused-vars
+  static async getMessage(data) {
     return `Please configure an error message for "${this.name}" by overriding "static async getMessage()"`;
   }
 
   /**
-   * @param {FormControl} formControl
+   * @param {HTMLElement} formControl
```

Comment:
Would it be possible to make this of type FormControlMixin?

Reply:
Would it make sense to eventually put {LionField} here? Since that's the most logic common denominator I think? Otherwise FormControlMixin would make sense yes.

Reply:
No, because fieldsets/formgroups don't inherit from LionField :)